diff -u -ru profile-sync-daemon-6.29.orig/common/profile-sync-daemon.in profile-sync-daemon-6.29/common/profile-sync-daemon.in
--- profile-sync-daemon-6.29.orig/common/profile-sync-daemon.in	2016-11-24 06:50:43.000000000 +0900
+++ profile-sync-daemon-6.29/common/profile-sync-daemon.in	2016-11-24 17:06:51.916630757 +0900
@@ -552,16 +552,18 @@
 }
 
 parse() {
-  psd_state=$(systemctl --user is-active psd)
-  resync_state=$(systemctl --user is-active psd-resync.timer)
-  [[ "$psd_state" = "active" ]] && psd_color="${GRN}" || psd_color="${RED}"
-  [[ "$resync_state" = "active" ]] && resync_color="${GRN}" || resync_color="${RED}"
-  echo -e " ${BLD}Systemd service is currently ${psd_color}$psd_state${NRM}${BLD}.${NRM}"
-  echo -e " ${BLD}Systemd resync-timer is currently ${resync_color}$resync_state${NRM}${BLD}.${NRM}"
-  [[ $OLFS -eq 1 ]] &&
-    echo -e "${BLD} Overlayfs v$OLFSVER is currently ${GRN}active${NRM}${BLD}.${NRM}" ||
-    echo -e "${BLD} Overlayfs technology is currently ${RED}inactive${NRM}${BLD}.${NRM}"
-  echo
+  if `which systemctl >/dev/null 2>&1`;then 
+    psd_state=$(systemctl --user is-active psd)
+    resync_state=$(systemctl --user is-active psd-resync.timer)
+    [[ "$psd_state" = "active" ]] && psd_color="${GRN}" || psd_color="${RED}"
+    [[ "$resync_state" = "active" ]] && resync_color="${GRN}" || resync_color="${RED}"
+    echo -e " ${BLD}Systemd service is currently ${psd_color}$psd_state${NRM}${BLD}.${NRM}"
+    echo -e " ${BLD}Systemd resync-timer is currently ${resync_color}$resync_state${NRM}${BLD}.${NRM}"
+    [[ $OLFS -eq 1 ]] &&
+      echo -e "${BLD} Overlayfs v$OLFSVER is currently ${GRN}active${NRM}${BLD}.${NRM}" ||
+      echo -e "${BLD} Overlayfs technology is currently ${RED}inactive${NRM}${BLD}.${NRM}"
+    echo
+  fi
   echo -e "${BLD}Psd will manage the following per ${BLU}${PSDCONF}${NRM}${BLD}:${NRM}"
   echo
   local browser
